src/DssSpell.t.base.sol

```diff
@@ -929,6 +985,115 @@ contract DssSpellTestBase is Config, DSTest, DSMath {
         assertEq(token.balanceOf(address(join)), 0);
     }
 
+    function checkCropCRVLPIntegration(
+        bytes32 _ilk,
+        CropJoinLike join,
+        ClipAbstract clip,
+        CurveLPOsmLike pip,
+        address _medianizer1,
+        address _medianizer2,
+        bool _isMedian1,
+        bool _isMedian2,
+        bool _checkLiquidations
+    ) public {
+        pip.poke();
+        hevm.warp(block.timestamp + 3601);
+        pip.poke();
+        spotter.poke(_ilk);
+
+        // Check medianizer sources
+        assertEq(pip.orbs(0), _medianizer1);
+        assertEq(pip.orbs(1), _medianizer2);
+
+        // Authorization
+        assertEq(join.wards(pauseProxy), 1);
+        assertEq(vat.wards(address(join)), 1);
+        assertEq(clip.wards(address(end)), 1);
```

Comment:
I think this check is redundant, as already present on L1015
```suggestion
```

Reply:
ah yes was wondering why that was deleted, removed